nbfetch/templates/confirm.html

```diff
@@ -0,0 +1,62 @@
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"
+      integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+
+  </head>
+  <body>
+    <div class="modal fade" id="confirmModal" tabindex="-1" role="dialog" aria-labelledby="confirmModalLabel"
+    aria-hidden="true">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="confirmModalLabel">Overwrite Existing Directory?</h5>
+        </div>
+        <div class="modal-body">
+          Directory "{{directory}}" already exists.  
+          <br><br>
+          Do you want to simply go to the current directory or 
+          do you want to overwrite the current contents, possibly erasing any changes you have made?
```

Comment:
@dtarb - Would you like to weigh in on the messaging?

Reply:
I suggest

`<h5 class="modal-title" id="confirmModalLabel">Directory Exists</h5>`

`<div class="modal-body">`
`There is a directory in Jupyterhub with the same name as the one being accessed from HydroShare.  `
`<\div>`
`<div class="modal-footer">`
`          <button id="overwrite" class="btn btn-danger" data-dismiss="modal">Overwrite Jupyterhub directory replacing it with files from HydroShare</button>`
 `         <button id="goto" class="btn btn-primary" data-dismiss="modal">Retain current Jupyterhub directory unchanged</button>`
`        </div>`

Reply:
You don't want to display the directory name?

I'm not a fan or large buttons with lots of text.  I'm having trouble making it work too.

![Selection_110](https://user-images.githubusercontent.com/6372515/56149224-12594580-5f7a-11e9-90ab-9e6008af59e8.png)

Reply:
Changes the title as suggested and made some other tweaks...
![Selection_111](https://user-images.githubusercontent.com/6372515/56150699-6fa2c600-5f7d-11e9-8c37-7aac552aaecb.png)